packages/client/ui/react-native-ui/README.md

```diff
@@ -0,0 +1 @@
+# `@crossmint/client-sdk-react-native-ui`
```

Comment:
I would call the folder just `react-native` (removing the `-ui`).

Reply:
done https://github.com/Crossmint/crossmint-sdk/pull/1020/commits/c421aa337197a3f07abfc31fe9857941dee8889e